src/combo.lua

```diff
@@ -16,6 +16,18 @@ function Combo.comboKeys(key)
     end
 end
 
+function Combo.checkCombo(key, keyInCombo)
+    keyEntry = {}
+    for i, keyInCombo in ipairs(Combo.keyList) do
+        if key == keyInCombo then
+            table.insert(keyEntry, #keyEntry + 1, keyInCombo)
+            if #keyEntry >= #keyInCombo then
+                love.graphics.print(Combo, Player.x, Player.y - 50)
+            end
+        end
+    end
+end
+
```

Comment:
Lets try to rewrite this..

```
Combo.fireballComboInput = {'a', 'f', 'h'}
Combo.comboInputs= {Combo.fireballComboInput}
function Combo.checkCombo()
    for i, comboInput in ipairs(Combo.comboInputs) do
        keyHistory = table.unpack(Combo.keyList, #Combo.keylist - #comboInput)
        love.graphics.print("comboInput", comboInput, "keyHistory", keyHistory)
        if keyHistory == comboInput then
            love.graphics.print(Combo, Player.x, Player.y - 50)
        end
    end
end
```

Reply:
yep

Reply:
Trying to get this to work. It looks like LOVE uses luaJIT, which supports lua 5.1 functionality (instead of 5.2) so that means it's unpack instead of table.unpack.

Reply:
I'm trying to understand unpack and getting results I don't expect still.
![Screenshot 2022-09-09 at 14 07 05](https://user-images.githubusercontent.com/3842383/189362560-bdd8c9c9-47a5-4a8e-adc5-ef373dd16c3f.png)